src/main/resources/application.yml

```diff
@@ -54,4 +54,6 @@ xpay:
 
 springdoc:
   swagger-ui:
-    path: /swagger-ui
\ No newline at end of file
+    path: /swagger-ui
+
+HOST_URL: ${AZURE_HOST_URL:http://localhost:${server.port}}
```

Comment:
@umberto-russo-nexi it's better if you call this property in a more self-explaining way, such as VPOS_BASE_URL or similar.
Also, just as a reminder, remember to add this parameter in the app-service's configuration map